new_dawn_server/users/api/resources.py

```diff
@@ -276,7 +277,8 @@ class Meta:
             'age': ALL_WITH_RELATIONS,
             'review_status': ALL_WITH_RELATIONS,
         }
-        queryset = Profile.objects.all()
+        profiles_count = Profile.objects.all().count()
+        queryset = Profile.objects.all().order_by('?')[:min(profiles_count,3)]
```

Comment:
Let's create a constant at the beginning of the file called MAX_PROFILE and use that here.